example/SpatialAudioLab.wlp

```diff
@@ -17,6 +17,18 @@
                 0.9999998,
                 1.0,
                 0.9999998
+            ],
+            "components": [
+                {
+                    "type": "wasd-controls"
```

Comment:
You should set the head to NonVrCamera, otherwise the mouse-look does not affect the wasd direction

Reply:
Also, I think the NonVrCamera has a teleport component attached that you might want to remove?

Reply:
it didnt have a teleport